src/sentry/static/sentry/app/components/button/index.jsx

```diff
@@ -187,7 +190,7 @@ const getColors = ({priority, disabled, borderless, theme}) => {
   `;
 };
 
-const StyledButton = styled(({busy, external, borderless, ...props}) => {
+const StyledButton = styled(({busy, external, borderless, disabled, ...props}) => {
```

Comment:
I'm fine with this since it's already here, but it might be nice to formally decide wether we prefer destructuring props away like this, or using `shouldForwardProps` to solve this problem. 

I lean towards the latter because (like in this example) the list can get kind of long and I don't think there is a ton of benefit to knowing what props are being filtered out, only what is going through. 

`shouldForwardProp` is also nice because it might prevent four or five more future trips to this component for the same problem.

That said. there is an argument for keeping things in simple es6 form when possible, rather than relying on mysterious imported functions. So I would be curious what other folks think.

Reply:
Would `shouldForwardProp` even work for this case since `disabled` is a valid DOM property?

Reply:
You have to explicitly exclude it with `shouldForwardProp`. Example:

https://github.com/getsentry/sentry/blob/7e41b9de5566d1fb7d0d3eda8791d73da9801f09/src/sentry/static/sentry/app/views/settings/components/forms/field/fieldLabel.jsx#L4

Reply:
that is a good point—maybe this isn't the best example, but it would be valid for the other three props.

Reply:
Updated